### DISCUSSION TITLE
请求添加瘟疫公司

### DISCUSSION THREAD

**Discussion by 972933728** [Gamuxorg/bbs#231] [投递和请求游戏]
游戏名称：瘟疫公司
游戏名称(英文)：Plague Inc
下载地址 :（如果有的话）：

**Comment by hyh19962008** [comment]
站内有的，请搜索一下

Best

________________________________

**Comment by 972933728** [comment]
> 站内有的，请搜索一下 Best
> […](#)
> ________________________________ From: 972933728 <notifications@github.com> Sent: Friday, February 14, 2020 12:23:23 PM To: Gamuxorg/bbs <bbs@noreply.github.com> Cc: Subscribed <subscribed@noreply.github.com> Subject: [Gamuxorg/bbs] 请求添加瘟疫公司 (#76) 游戏名称：瘟疫公司 游戏名称(英文)：Plague Inc 下载地址 :（如果有的话）： ― You are receiving this because you are subscribed to this thread. Reply to this email directly, view it on GitHub<#76?email_source=notifications&email_token=AFRK4UOPXM3ITPM6N3AYOCDRCYMDXA5CNFSM4KVAOFFKYY3PNVWWK3TUL52HS4DFUVEXG43VMWVGG33NNVSW45C7NFSM4INOV3FA>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AFRK4ULUZO3R6LXWGV3BHFDRCYMDXANCNFSM4KVAOFFA>.

站内没有，我找过了。还有那个搜索请维修一下，不能用。站内很多资源都失效了。

**Comment by shouhuanxiaoji** [comment]
您好 由于正在全力编写新版本网站，因此旧版本的一些问题不再修复。可暂时到de.linuxgame.cn寻找下载游戏

**Comment by hyh19962008** [comment]
已添加
